qiita_pet/handlers/analysis_handlers.py

```diff
@@ -141,14 +142,17 @@ def get(self, analysis_id):
             jobres[jobject.datatype].append((jobject.command[0],
                                              jobject.results))
 
-        dropped = {}
         dropped_samples = analysis.dropped_samples
+        dropped = defaultdict(list)
         if dropped_samples:
```

Comment:
@adamrp I think you suggested that this `if` statement can be removed. However, `analysis.dropped_samples` returns `None` if there are no dropped samples, instead of an empty list. This will make the call to `viewitems` to fail because of an AttributeError.

I think this should not block this PR and we should open a new issue for modifying `analysis.dropped_samples` so it returns consistent types (list). Do you agree?